README.md

```diff
@@ -263,6 +263,46 @@ An instance of `DidEndpoint` contains the service details, i.e., service ID, a s
 
 > The validity and authenticity of the information stored on the KILT blockchain are guaranteed by the same blockchain, which verifies that all DID management operations other than creation are signed by the DID authentication key, as described in the section relative to each operation.
 
+#### Representation of Public Key Types on the DID Document
+
+Currently the use of four different types of public keys as [Verification Methods](https://w3c.github.io/did-core/#verification-methods) are supported by a full DID; light DIDs support a subset of these (see [Light DIDs](#light-dids) section for details).
+Keys type is indicated in form of an enum for both full DIDs and light DIDs.
+By reference implementations of a DID resolver compliant with the specification, keys are represented as follows on the DID document.
+
+Conforming with [DID specifications](https://w3c.github.io/did-core/#verification-methods) these Verification Methods are JSON objects containing the properties `id`, `type`, and `controller`.
+All Verification Methods on a DID Document returned from these resolvers have an additional property `publicKeyBase58` containing the public key material in base58 encoding using the bitcoin alphabet.  
```

Comment:
I am not sure I fully understand the meaning of this sentence. Does it say that all public keys are resolved with their base58 representation? If that is the case, it is not true since they could use alternative representation formats such as JWK.

Reply:
If, on the other hand, it refers to our resolver, then it makes sense.

Reply:
It refers to our (reference) implementation, which returns keys base58 encoded. Hope that's reasonably clear?

Reply:
Unfortunately I do not think it is, as far as I have understood. It seems to be referring to resolvers in general, not to resolvers compliant with the KILT DID method.

Reply:
I can see it is confusing... it's really hard for me to express this right, which comes from me trying to avoid making the key representation part of the spec (which would mean that we require all implementations to represent them this way). But this way it's these descriptions essentially document some (reference) implementation and thus do not really seem to fit here.